### PR TITLE
updating CCS instances according to latest change in CCS schema(s)

### PR DESCRIPTION
**Pull request by lzehl**
This PR corrects the existing instances for the CommonCoordinatSpace to the latest update in SANDS where the CCS are established as Research Products and Research Products Versions. 

The first commit prepared the split in the instances into CCS and CCS Versions and updated the CCS instances already. 

The "second" commit provides the updated instances for the CCS Versions.

### CODE REVIEW

**Comment by lzehl** [comment]
@aeidi89 we can have a look at this one together. We need to check the provided information (especially axes orientations for each CCSV). Missing information we can add later on in different PRs. The instances of this PR have to be correct for the cleanup (so the cleanup PRs should go through first; so that is easier to see if we missed an update in the instances).

File: instances/atlas/commonCoordinateSpace/AMB-CCF.jsonld [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+			"@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpaceVersion/AMB-CCF_v3"
+		},
+		{
+			"@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpaceVersion/AMB-CCF_v2"
+		},
+		{
+			"@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpaceVersion/AMB-CCF_v1"
+		}
+	],
+	"homepage": "http://atlas.brain-map.org/",
```
Comment (aeidi89) [inline instances/atlas/commonCoordinateSpace/AMB-CCF.jsonld]: The link that is provided is the nicest one for getting started straight away. However, if you click the title on that page you're re-directed here: https://portal.brain-map.org/ (I think this is the true home page)

If you want information about the reference spaces I think this web page might be better: https://allensdk.readthedocs.io/en/latest/reference_space.html

File: instances/atlas/commonCoordinateSpace/BigBrain.jsonld [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	"fullName": "BigBrain Whole-Brain Model",
+	"hasVersion": [
+		{
+			"@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpaceVersion/BigBrain_2015"
+		}
+	],
+	"homepage": "https://bigbrainproject.org/",
+	"howToCite": null,
+  "ontologyIdentifier": null,
+	"shortName": "BigBrain Model"
```
Comment (aeidi89) [inline instances/atlas/commonCoordinateSpace/BigBrain.jsonld:21]: Is the short name for the BigBrain common coordinate space 'BigBrain Model'?
Would it not be more logical if it was 'BigBrain Space'?

File: instances/atlas/commonCoordinateSpace/MEBRAINStemplate.jsonld [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	"fullName": "MEBRAINS population-based monkey brain template",
+	"hasVersion": [
+		{
+			"@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpaceVersion/MEBRAINStemplate_v1.0"
+		}
+	],
+	"homepage": null,
+	"howToCite": null,
+  "ontologyIdentifier": null,
+	"shortName": "MEBRAINS brain template"
```
Comment (aeidi89) [inline instances/atlas/commonCoordinateSpace/MEBRAINStemplate.jsonld:21]: Similar comment as above: would it be more logical to have a short name 'MEBRAINS Space'?

File: instances/atlas/commonCoordinateSpace/MEBRAINStemplate.jsonld [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+		"@vocab": "https://openminds.ebrains.eu/vocab/"
+	},
+	"@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpace/MEBRAINStemplate",
+	"@type": "https://openminds.ebrains.eu/sands/CommonCoordinateSpace",
+	"abbreviation": "MEBRAINStemplate",
+	"author": null,
+	"custodian": null,
+	"description": "The 'MEBRAINS population-based monkey brain template' is a multi-subject based, multi-modal, volume and surface brain template for macaque monkeys.",
+	"digitalIdentifier": null,
+	"fullName": "MEBRAINS population-based monkey brain template",
```
Comment (aeidi89) [inline instances/atlas/commonCoordinateSpace/MEBRAINStemplate.jsonld:12]: Is this the full name of the atlas or the space?

File: instances/atlas/commonCoordinateSpace/BigBrain.jsonld [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+		"@vocab": "https://openminds.ebrains.eu/vocab/"
+	},
+	"@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpace/BigBrain",
+	"@type": "https://openminds.ebrains.eu/sands/CommonCoordinateSpace",
+	"abbreviation": "BigBrain",
+	"author": null,
+	"custodian": null,
+	"description": "The 'BigBrain Whole-Brain Model' is a 3D reconstruction of a human brain in extremely high resolution.",
+	"digitalIdentifier": null,
+	"fullName": "BigBrain Whole-Brain Model",
```
Comment (aeidi89) [inline instances/atlas/commonCoordinateSpace/BigBrain.jsonld:12]: Is this the name of the atlas or the common coordinate space?

File: instances/atlas/commonCoordinateSpace/MNI-Colin27.jsonld [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+	"hasVersion": [
+		{
+			"@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpaceVersion/MNIColin27_1998"
+		},
+		{
+			"@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpaceVersion/MNIColin27_2008"
+		}
+	],
+	"homepage": "https://www.mcgill.ca/bic/software/tools-data-analysis/anatomical-mri/atlases",
+	"howToCite": "Holmes CJ, Hoge R, Collins L, Woods R, Toga AW, and Evans AC; 'Enhancement of MR images using registration for signal averaging.'; J Comput Assist Tomogr; 1998 Mar-Apr; 22(2):324–33. [doi: 10.1097/00004728-199803000-00032](http://dx.doi.org/10.1097/00004728-199803000-00032)\nAubert-Broche B, Evans AC, and Collins DL; 'A new improved version of the realistic digital brain phantom'; NeuroImage; 2006 Aug;32(1):138–45. [doi: 10.1016/j.neuroimage.2006.03.052](https://doi.org/10.1016/j.neuroimage.2006.03.052)",
```
Comment (aeidi89) [inline instances/atlas/commonCoordinateSpace/MNI-Colin27.jsonld:22]: In "howToCite": "Holmes CJ, Hoge R, Collins L, Woods R, Toga AW, and Evans AC; 'Enhancement of MR images using registration for signal averaging.'; J Comput Assist Tomogr; 1998 Mar-Apr; 22(2):324–33. [doi: 10.1097/00004728-199803000-00032](http://dx.doi.org/10.1097/00004728-199803000-00032)\nAubert-Broche B, Evans AC, and Collins DL; 'A new improved version of the realistic digital brain phantom'; NeuroImage; 2006 Aug;32(1):138–45. [doi: 10.1016/j.neuroimage.2006.03.052](https://doi.org/10.1016/j.neuroimage.2006.03.052)",


\nAubert-Broche B

@lzehl, Is this a minor error or should it be written like this?

File: instances/atlas/commonCoordinateSpace/MNI-Colin27.jsonld [hunk 7]
```diff
@@ -0,0 +1,28 @@
+{
+	"@context": {
+		"@vocab": "https://openminds.ebrains.eu/vocab/"
+	},
+	"@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpace/MNI-Colin27",
+	"@type": "https://openminds.ebrains.eu/sands/CommonCoordinateSpace",
+	"abbreviation": "Colin 27",
```
Comment (aeidi89) [inline instances/atlas/commonCoordinateSpace/MNI-Colin27.jsonld]: Should the space between 'Colin' and '27' be removed?

@lzehl

File: instances/atlas/commonCoordinateSpaceVersion/AMB-CCF_v3-RAS.jsonld [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+	"custodian": null,
+	"defaultImage": null,
+	"description": null,
+	"digitalIdentifier": null,
+	"fullDocumentation": null,
+	"fullName": "Allen Mouse Brain Common Coordinate Framework",
+	"funding": null,
+	"homepage": "https://portal.brain-map.org/",
+	"howToCite": "Wang Q, Ding S-L, Li Y, et al.; 'The Allen Mouse Brain Common Coordinate Framework: A 3D Reference Atlas.'; Cell; May 2020; 181(4):936-953.e20. [doi: 10.1016/j.cell.2020.04.007](https://doi.org/10.1016/j.cell.2020.04.007)",
+	"isAlternativeVersionOf": null,
```
Comment (aeidi89) [inline instances/atlas/commonCoordinateSpaceVersion/AMB-CCF_v3-RAS.jsonld]: @lzehl Shouldn't this have AMB-CCF_v3?

File: instances/atlas/commonCoordinateSpaceVersion/WHSSD_v1.jsonld [hunk 9]
```diff
@@ -0,0 +1,41 @@
+{
+	"@context": {
+		"@vocab": "https://openminds.ebrains.eu/vocab/"
+	},
+	"@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpaceVersion/WHSSD_v1",
+	"@type": "https://openminds.ebrains.eu/sands/CommonCoordinateSpaceVersion",
+	"abbreviation": "WHSSD",
+	"accessibility": null,
+	"anatomicalAxesOrientation": {
+		"@id": "https://openminds.ebrains.eu/instances/anatomicalAxesOrientation/RAS"
```
Comment (aeidi89) [inline instances/atlas/commonCoordinateSpaceVersion/WHSSD_v1.jsonld]: This should be ALS orientation

**Review by aeidi89 — COMMENTED** [review]
A few comments needs checking and one change, otherwise looks good

**Review by aeidi89 — APPROVED** [review]
Looks good

**Pull request merged**